plugin.video.spurs-tv/changelog.txt

```diff
@@ -1,87 +1,90 @@
-2.8.0
+v2.9.0 (2016-07-31)
+ - Added stadium build live streams
+ - Fixed Tour 2016 category
+
+v2.8.0 (2016-03-19)
  - Fixed YouTube playlists
  - Updated fanart
 
-2.7.0
+v2.7.0 (2016-02-28)
  - Kodi Krypton is now a requirement
  - Use Rollbar for automatically reporting errors
```

Comment:
You may want to make the Rollbar reporting controllable by a user option - some user's may not want this.